das/daser.go

```diff
@@ -222,8 +229,11 @@ func (d *DASer) catchUp(ctx context.Context, job *catchUpJob) (int64, error) {
 
 		startTime := time.Now()
 
-		err = d.da.SharesAvailable(ctx, h.DAH)
+		ctxWithSpan, span := otel.Tracer(tracerName).Start(ctx, "catchUp")
```

Comment:
Same here. We create multiple tracers with the same name, which conflict with each other. Instead, there should be one global.